src/understand-tidb/plugin.md

```diff
@@ -1 +1,63 @@
 # Plugin
+
+The plugin API allows TiDB to be extended with new features such as audit logging or IP allow/deny listing.
+
+Sample code is provided for a basic audit logging plugin at [`./tidb/plugin/conn_ip_example/`](https://github.com/pingcap/tidb/tree/master/plugin/conn_ip_example). For an example on compiling TiDB and this plugin:
```

Comment:
```suggestion
Sample code is provided for a basic audit logging plugin at [`plugin/conn_ip_example/`](https://github.com/pingcap/tidb/tree/master/plugin/conn_ip_example). For an example on compiling TiDB and this plugin:
```

I'd prefer use relative path to root path of the repository.

Reply:
I've incorporated this, thanks!